bitcoin/BRSyncManager.c

```diff
@@ -1353,7 +1355,9 @@ BRClientSyncManagerUpdateTransactions (BRClientSyncManager manager) {
                                                       (const char **) addresses,
                                                       addressCount,
                                                       begBlockNumber,
-                                                      endBlockNumber,
+                                                      (BRClientSyncManagerScanStateIsFullScan (&manager->scanState)
```

Comment:
Same comment re:moving to `BRClientSyncManagerScanStateInit`/`BRClientSyncManagerScanStateGetEndBlockNumber`.